autosklearn/pipeline/components/data_preprocessing/rescaling/power_transformer.py

```diff
@@ -0,0 +1,32 @@
+from autosklearn.pipeline.constants import DENSE, UNSIGNED_DATA, INPUT
+from autosklearn.pipeline.components.data_preprocessing.rescaling.abstract_rescaling \
+    import Rescaling
+from autosklearn.pipeline.components.base import AutoSklearnPreprocessingAlgorithm
+
+
+class PowerTransformerComponent(Rescaling, AutoSklearnPreprocessingAlgorithm):
+    def __init__(self, random_state):
+        from sklearn.preprocessing import PowerTransformer
+        self.preprocessor = PowerTransformer(copy=False)
+
+    @staticmethod
+    def get_properties(dataset_properties=None):
+        return {'shortname': 'PowerTransformer',
+                'name': 'PowerTransformer',
+                'handles_missing_values': False,
+                'handles_nominal_values': False,
+                'handles_numerical_features': True,
+                'prefers_data_scaled': False,
+                'prefers_data_normalized': False,
+                'handles_regression': True,
+                'handles_classification': True,
+                'handles_multiclass': True,
+                'handles_multilabel': True,
+                'handles_multioutput': False,
+                'is_deterministic': True,
+                # TODO find out of this is right!
```

Comment:
This can be removed as you figured this out, right?

Reply:
Hi Matthias, this is what all other transformations say. Should we remove this everywhere?

Reply:
Yes, that would be great. Maybe this can be done in conjunction with #749?